token_filter.info.yml

```diff
@@ -3,3 +3,6 @@ name: 'Token Filter'
 description: 'Allows token values to be used as filters.'
 core: 8.x
 version: VERSION
+dependencies:
+  - filter
+  - token
```

Comment:
I disagree hear for three primary reasons:

1. Without token as a dependency the module can not reach feature parity with the Drupal 7 release, and a port that removes functionality is not a port, nor is it a re-write, but an interpretation of a portion of the functionality of an existing module.

2. Token filter without a Token tree is entirely un-usable by your average content creator, who is the target market for the functionality.

3. The Token module is already a dependency of the Token Filter module in D7, and therefore retaining that dependency is entirely reasonable and expected by the existing users. Removing it isn't a bad thing, but neither is retaining it, and the only thing removing it achieves is the reduction of functionality and usability of the module.